content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -394,44 +410,28 @@ Please also note the changed `conmon_cgroup`, which has to be set to the value
 cgroup driver configuration of the kubelet (usually done via kubeadm) and CRI-O
 in sync.
 
-### Docker
+### Docker Engine {#docker}
 
-1. On each of your nodes, install the Docker for your Linux distribution as per 
-[Install Docker Engine](https://docs.docker.com/engine/install/#server). 
-You can find the latest validated version of Docker in this 
-[dependencies](https://git.k8s.io/kubernetes/build/dependencies.yaml) file.
+Docker Engine is the container runtime that started it all. Formerly known just as Docker,
+this container runtime is available in various forms.
+[Install Docker Engine](https://docs.docker.com/engine/install/) explains your options
+for installing this runtime.
 
-2. Configure the Docker daemon, in particular to use systemd for the management of the container’s cgroups.
+Docker Engine is directly compatible with Kubernetes {{< skew currentVersion >}}, using the deprecated `dockershim` component. For more information
```

Comment:
Should we explicitly mention that it's compatible with Kubernetes <= 1.23 instead of using currentVersion?

Reply:
This is the most common way we write this kind of detail that can change between versions. What we sometimes add is text to the effect of “if you are using a different version of Kubernetes, consult the documentation for that version”.

I could add that.

Reply:
@sftim I thought that '{{< skew currentVersion >}}' will be replaced with current version, e.g. when 1.24 is out this sentence would look like "Docker Engine is directly compatible with Kubernetes 1.24, using the deprecated `dockershim` component.", which is not correct as dockershim will most probably be removed in 1.24.

Reply:
I could hard-code v1.23 because I know that the `dev-1.24` branch is going to further revise this text. I do think this is OK to merge as is.

Reply:
makes sense.

Reply:
@sftim I think it's okay to merge this to main as is. I assume you're going to follow with a change on 1.24 branch?